.travis.yml

```diff
@@ -6,9 +6,21 @@ scala:
 jdk:
   - oraclejdk8
   - openjdk11
-  
-script:
-  - sbt clean coverage test coverageReport
+
+stages:
+  - name: test
+  - name: release
+    if: (branch = master AND type = push) OR (tag IS present)
```

Comment:
How does this plugin know how to update the version number? or will this override the current published snapshot version?

Reply:
Oke good to know!